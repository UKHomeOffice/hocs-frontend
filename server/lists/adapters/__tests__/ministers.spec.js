const ministerAdapter = require('../ministers');

describe('Minister Adapter', () => {
    it('should transform and sort minister data', async () => {
        const mockData = {
            ministers: [
                { label: 'Minister A', value: 1 },
                { label: 'Minister C', value: 3 },
                { label: 'Minister B', value: 2 }
            ]
        };

        const results = await ministerAdapter(mockData);
        expect(results).toBeDefined();
        expect(results).toMatchSnapshot();
    });
});