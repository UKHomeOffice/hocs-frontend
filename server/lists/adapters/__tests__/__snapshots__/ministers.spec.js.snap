// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Minister Adapter should transform and sort minister data 1`] = `
Array [
  Object {
    "label": "Minister A",
    "value": 1,
  },
  Object {
    "label": "Minister B",
    "value": 2,
  },
  Object {
    "label": "Minister C",
    "value": 3,
  },
]
`;
